docs/topics/db/queries.txt

```diff
@@ -1110,6 +1110,8 @@ that combine both a normal query and a negated (``NOT``) query::
 
     Q(question__startswith='Who') | ~Q(pub_date__year=2005)
 
+You may also create a ``Q`` with no arguments, which applies no lookups.
```

Comment:
Outside the context of the mailing list thread (and in the context of these examples), this seems like a strange thing to do.